TM1py/Services/ElementService.py

```diff
@@ -275,3 +276,53 @@ def get_members(element):
 
         get_members(consolidation_tree)
         return members
+
+    def execute_set_mdx(self, mdx,
+                        top_records=None,
+                        member_properties = ['Name', 'Weight'],
+                        parent_properties = ['Name', 'UniqueName'],
+                        element_properties = ['Type', 'Level']):
```

Comment:
This is a common gotcha! https://docs.python-guide.org/writing/gotchas/

It's safer to use a `tuple` here, as it is immutable, or to use `None` as a default argument and assign the list to the argument inside the fucntion.